app/src/main/res/anim/slide_up.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="400">
+    <translate
+        android:fromYDelta="50%p"
+        android:toYDelta="0%p" />
```

Comment:
Let's be consistent with empty line usage for it works for better readability - this one doesn't seem the same as slide_down.xml.